perceval/backends/core/finosmeetings.py

```diff
@@ -0,0 +1,273 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018-2019 Fintech Open Source Foundation
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA.
+#
+# Authors:
+#     Maurizio Pillitu <maoo@finos.org>
+#
+
+import base64
+import logging
+import datetime
+import tempfile
+
+import csv
+from grimoirelab_toolkit.datetime import datetime_utcnow
+
+from ...backend import (Backend,
+                        BackendCommand,
+                        BackendCommandArgumentParser)
+from ...client import HttpClient
+
+CATEGORY_ENTRY = "finos-meeting-entry"
+
+logger = logging.getLogger(__name__)
+
+
+class FinosMeetings(Backend):
+    """FinosMeetings backend for Perceval.
+
+    This class retrieves the entries from a CSV file.
+    To initialize this class the CSV file path must be provided.
+    The `file_path` will be set as the origin of the data.
+
+    :param uri: URI pointer to FINOS meeting data
+    :param csv_header: Columns included in the CSV file
+    :param separator: CV separator char
+    :param date_formats: Comma-separated list of date formas to use to extract the timestamp of a CSV entry
+    :param skip_header: 'true' if the first CSV row contains the column header
+    :param id_columns: the columns that compose the ID hash
+    :param date_column: the column containing the date for metadata_updated_on
+    """
+    version = '0.0.1'
+
+    CATEGORIES = [CATEGORY_ENTRY]
+
+    separator = ','
+
+    def __init__(self, uri, csv_header, separator, date_formats, skip_header, id_columns, date_column, tag=None, archive=None):
+        super().__init__(uri)
```

Comment:
`tag` and `archive` should be given here and not set below. As in: https://github.com/chaoss/grimoirelab-perceval/blob/master/perceval/backends/core/git.py#L74